packaging/conda/meta.yaml

```diff
@@ -36,16 +36,57 @@ requirements:
   run:
     - {{ mpi }}
 
+# requirements:
+#   run:
+#     - khiops-core
+#     - kni
+#     - kni-transfer
+
 outputs:
   - name: khiops-core
+    version: {{ os.environ.get('KHIOPS_VERSION') }}
+    requirements:
+      build:
+        - cmake
+        - ninja
+        - {{ compiler('cxx') }}
+      host:
+        - {{ mpi }}
+      run:
+        - {{ mpi }}
+    files:
+      - bin/MODL*
     test:
       commands:
         - MODL -v
         - MODL_Coclustering -v
+  - name: kni
+    version: {{ os.environ.get('KHIOPS_VERSION') }}
+    files: 
+      - bin/libKhiopsNativeInterface* # [linux or osx]
+      - bin/KhiopsNativeInterface.dll # [win]
+    requirements:
```

Comment:
I would add a comment here that KNI does not need  `khiops-core` as a runtime dependency.

Reply:
done